src/Utility/MultipartFormDataBinaryContent.cs

```diff
@@ -161,19 +180,19 @@ public override void WriteTo(Stream stream, CancellationToken cancellationToken
         // TODO: polyfill sync-over-async for netstandard2.0 for Azure clients.
         // Tracked by https://github.com/Azure/azure-sdk-for-net/issues/42674
 
-#if NET6_0_OR_GREATER
+#if NET
```

Comment:
Curious: why is this pattern preferable to using `NET6_0_OR_GREATER`?

Reply:
Just for consistency and brevity. It's fine to want to be precise, but then NET6_0_OR_GREATER is arguably "incorrect" here, as the guarded API was introduced earlier than that; it's just guarding it with NET6_0_OR_GREATER than because that's the oldest version of core the library targets, but then you can also just use the simpler/shorter "NET", which has the same effect. I can put it back if you'd prefer, though.

Reply:
I don't feel strongly, and overall I'm happy to learn.  @KrzysztofCwalina, do you have an opinion?  Our SDK libraries typically only target netstandard2.0 and net6.0, so it's correct in that context, and if we stick with NET6_0_OR_GREATER, it leaders to fewer preprocessor symbols in client code, but I'm open to revisiting what we standardize around with new SCM-based libraries.

Reply:
I don't feel strongly either way, but: a) I prefer being precise with the minimum version we need, i.e. 5.0+, b) I slightly prefer being explicit, i.e. NET_5_0_OR_GREATER; it makes it easier to read the code.